components/automate-cs-oc-erchef/habitat/default.toml

```diff
@@ -38,6 +40,10 @@ reindex_batch_size=10
 reindex_sleep_min_ms=50
 reindex_sleep_max_ms=2000
 reindex_item_retries=3
+search_provider="elasticsearch"
```

Comment:
should the default change to opensearch here?

Reply:
This is the default value for `search_provider` which can be updated with `opensearch` while deployment by patching.